dll/win32/ws2_32/src/startup.c

```diff
@@ -105,6 +141,75 @@ WSACleanup(VOID)
     return ErrorCode;
 }
 
+/* WSACallThread will call a WSA function in an other thread and returns the result  */
+
+INT
+WINAPI
+WSACallThread( IN int funcId, IN LPVOID arg1, IN LPVOID arg2)
+{
+    int lastError;
+    int returnvalue;
+
+   /* wait until the Thread is ready to accept commands and lock it */
+    WsCreateWSAThreadSynchronization();
+
+    /* set WSAThread parameters */
+    WSAThreadCallId = funcId;
+    WSAThreadArg1 = arg1;
+    WSAThreadArg2 = arg2;
+
+    /* tell WSAThread the parameters have been set and it can preform itś task */
```

Comment:
Suspicious character "ś".

Reply:
it took me a while to find the character, I think i meant it's  instead of its, typo :+1:  Thanks!